### DISCUSSION TITLE
汽车

### DISCUSSION THREAD

**Discussion by getchangethat** [share121/inter-knot#433] [其他]
谁知道二十十万左右，有啥驾驶不错的车车
![14ff4da63e3737f1](https://github.com/user-attachments/assets/e8d99b03-30de-4f48-abd8-60afeb434f78)

**Comment by Ailian-Qiao** [comment]
小米

**Comment by sunhexuan** [comment]
凯迪拉克ct5

**Comment by MocoTEL** [comment]
f代的宝马可以考虑考虑

**Comment by AKT505** [comment]
要油车还是电车

**Comment by leonwangls** [comment]
今年9-12月还有国家的3000亿消费补贴，可以看看新能源
